specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/config/proxy.when.client.topic.space.yaml

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+---
+name: test
+bindings:
+  mqtt0:
+    type: mqtt-kafka
+    kind: proxy
+    options:
+      topics:
+        sessions: mqtt-sessions
+        messages: mqtt-messages
+        retained: mqtt-retained
+      clients:
+        - /clients/{identity}/#
+    routes:
+      - when:
+        - publish:
+            - topic: /clients/#
+        with:
+          messages: mqtt-clients
+        exit: kafka0
+      - when:
+          - subscribe:
+              - topic: /clients/#
+        with:
+          messages: mqtt-clients
+        exit: kafka0
+      - when:
+          - subscribe:
+              - topic: /sensor-clients/#
+        with:
+          messages: mqtt-devices
+        exit: kafka0
+      - when:
+          - subscribe:
+              - topic: /device-clients/#
```

Comment:
Let's move second `subscribe` alongside `subscribe` above to avoid re-stating `with` and `exit`.